mlflow/spark.py

```diff
@@ -39,6 +41,47 @@ def log_model(spark_model, artifact_path, conda_env=None, jars=None):
                      jars=jars, conda_env=conda_env)
 
 
+def _tmp_path(len):
+    return ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(len))
+
+
+class _DFS:
```

Comment:
Maybe we can call this `_HadoopFileSystem`, since that's what it is shadowing? Could we also add a comment on the purpose of this class?